tests/in/binding-arrays.wgsl

```diff
@@ -63,6 +63,7 @@ fn main(fragment_in: FragmentIn) -> @location(0) vec4<f32> {
     v4 += textureLoad(texture_array_unbounded[non_uniform_index], pix, 0);
 
     u1 += textureNumLayers(texture_array_2darray[0]);
+    u1 += textureNumLayers(texture_array_2darray[0u]);
```

Comment:
This is indexing a `binding_array` which I think is not what you intended.

Reply:
Done.